src/applications/vaos/tests/appointment-list/components/FutureAppointmentsList.video.unit.spec.jsx

```diff
@@ -865,6 +866,15 @@ describe('VAOS <FutureAppointmentsList> calendar ics file format', () => {
       (_, node) => node.textContent === 'VA Video Connect at a VA location',
     );
 
+    // wait for facilities fetch
+    await waitFor(() => {
```

Comment:
Normally we would wait for something on the page to appear, like the facility name. That makes the test less dependent on the specifics of the fetch calls happening.